src/jit/gentree.h

```diff
@@ -940,6 +940,9 @@ struct GenTree
 
 #define GTF_ARRLEN_ARR_IDX 0x80000000 // GT_ARR_LENGTH -- Length which feeds into an array index expression
 
+#define GTF_LIST_AGGREGATE 0x80000000 // GT_LIST -- Indicates that this list should be treated as an
```

Comment:
> These GT_LIST nodes are need by LIR only when we have TYP_STRUCT argument that is passed on the stack.

This is a good explanation of the specific need. I do want to point out that for the sake of uniformity, all lists that represent arguments that may be passed in multiple registers are marked as aggregate arguments. This property is used e.g. by the operand iterator.